lib/soapy_cake/admin_addedit.rb

```diff
@@ -173,6 +172,15 @@ def edit_campaign(opts)
 
     private
 
+    def addedit_offer_tier(add_edit_option, opts)
+      require_params(opts, %i(offer_id tier_id price_format_id offer_contract_id status_id))
+
+      opts = opts.merge(redirect_offer_contract_id: -1, add_edit_option: add_edit_option)
```

Comment:
In order to create a new object instead of changing the object that gets passed in directly. This fell on my feet when I re-tried the call with the same options. The options where already changed in-place the first time, so the second call failed.